src/EspressoNitroTEEVerifier.sol

```diff
@@ -1,96 +1,70 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import {NitroValidator} from "@nitro-validator/NitroValidator.sol";
-import {LibBytes} from "@nitro-validator/LibBytes.sol";
-import {LibCborElement, CborElement, CborDecode} from "@nitro-validator/CborDecode.sol";
-import {CertManager} from "@nitro-validator/CertManager.sol";
 import "@openzeppelin/contracts/access/Ownable2Step.sol";
 import {IEspressoNitroTEEVerifier} from "./interface/IEspressoNitroTEEVerifier.sol";
+import {
+    INitroEnclaveVerifier,
+    VerifierJournal,
+    ZkCoProcessorType
+} from "aws-nitro-enclave-attestation/interfaces/INitroEnclaveVerifier.sol";
 
 /**
  * @title  Verifies quotes from the AWS Nitro Enclave (TEE) and attests on-chain
- * @notice Contains the logic to verify an attestation and signature from the TEE and attest on-chain. It uses the NitroValidator contract
- *         from `base` to verify the quote. Along with some additional verification logic.
- *         (https://github.com/base/nitro-validator)
- * The code of this contract is inspired from SystemConfigGlobal.sol
- * (https://github.com/base/op-enclave/blob/main/contracts/src/SystemConfigGlobal.sol)
+ * @notice Contains the logic to verify zk proof of the attestation on-chain. It uses the EspressoNitroTEEVerifier contract
+ *         from `automata` to verify the proof.
  */
-contract EspressoNitroTEEVerifier is NitroValidator, IEspressoNitroTEEVerifier, Ownable2Step {
-    using CborDecode for bytes;
-    using LibBytes for bytes;
-    using LibCborElement for CborElement;
-
+contract EspressoNitroTEEVerifier is IEspressoNitroTEEVerifier, Ownable2Step {
```

Comment:
do the consumers of the this contract have the ability to set a new EspressoNitroTEEVerifier in the event we ever have to deploy a new version?